Sources/MintKit/PackageReference.swift

```diff
@@ -17,12 +17,20 @@ public class PackageReference {
         let repo: String
         let version: String
         if packageParts.count == 3 {
-            repo = [packageParts[0], packageParts[1]].joined(separator: "@")
-            version = packageParts[2]
+            if packageParts[0].contains("ssh://") {
+                repo = [packageParts[0], packageParts[1]].joined(separator: "@")
+                version = packageParts[2]
+            } else {
+                repo = [packageParts[0], packageParts[1]].joined(separator: "@")
```

Comment:
Unless I'm missing something, the if and else seem to be doing the same thing here

Reply:
You're right, I've fixed it now